### PR TITLE
Feature/19

### PR DESCRIPTION
**Pull request by huangs08**

### CODE REVIEW

File: bike-erp/backend/src/main/helpers/db.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import mysql from 'mysql2';
+
+const db = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'mysqlpw',
```
Comment (nasaku898) [inline bike-erp/backend/src/main/helpers/db.ts:6]: could leave the password empty such that we can input our own pwd

**Review by nasaku898 — APPROVED** [review]
Great setup on the database!

**Review by Mnx458 — APPROVED** [review]
Version's are correct

File: bike-erp/backend/src/main/helpers/db.ts (hunk 1 above)
Comment (huangs08) [reply bike-erp/backend/src/main/helpers/db.ts]: Done!

**Pull request merged**